### ISSUE TITLE
how to run achillies test generator

### ISSUE THREAD

**Issue by zhaolida98** [secure-software-engineering/achilles-benchmark-depscanners#16]
Hi, I am trying to run the generator. However, I didn't find any detailed instructions.
I am using java 1.8 under ubuntu 18.04, and when I try to run the generator-1.6.jar by using `java -jar generator-1.6.jar` I got 
![image](https://user-images.githubusercontent.com/28668213/141996211-56fabbb7-5adc-4ad5-9214-30a1004336d5.png)
Then I try to rebuild the code using java 11, after resolving all missing deps (JavaFX is removed from java 11), I still can't get it right
![image](https://user-images.githubusercontent.com/28668213/141996404-55ea3c6e-f7ad-4c42-bf42-e46c0d621a80.png)
it ask me for the runtime components. 
Then I tried several method including add `--module-path' and '--add-modules' 
or directly add JavaFX plugin in pom file and run ' mvn compile javafx:run'
None of them works for me.
Could you please provide us the detailed instructions to run the generator? Including your OS info, java version.

Thanks!

**Comment by anddann** [comment]
Thanks @zhaolida98 for raising that question. 
What version of java are you using?

When I run  `java -jar generator-1.6.jar`the application is executed.

```
$ java -version
java version "1.8.0_301"
Java(TM) SE Runtime Environment (build 1.8.0_301-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.301-b09, mixed mode)
```

I'll try it out on different versions the next days

**Comment by zhaolida98** [comment]
Thanks a lot! With your new pr, I am able to build a new jar, now the box successfully pop up. However, I've tried several times, and I always met the downloading problem
![image](https://user-images.githubusercontent.com/28668213/142416367-3e5a824c-d853-4c71-8375-a376d466f73e.png)
Am I missing any other settings?

**Comment by anddann** [comment]
The maven url changed, I've fixed it in the PR.

**Comment by zhaolida98** [comment]
Thanks for your help, now the test cases can be properly generated. However, I can generate the test case only once a time. When I try to generate the second case in the same session, it will crash by showing 
![image](https://user-images.githubusercontent.com/28668213/142553826-e0aab5d9-34c0-4a40-983e-fcea6fe5f828.png)
and the full stack track is:
![image](https://user-images.githubusercontent.com/28668213/142553874-93c08575-22cc-4f42-a953-1c9f6c1be14d.png)

This problem does not really affect the test generating function, only about efficiency. Except that, everything else works perfectly, thank you again!

**Comment by anddann** [comment]
Thanks for pointing out the issue, I'll take a look over the weekend.
Can you please quickly describe the steps to reproduce?

**Comment by anddann** [comment]
Also it would be nice, if you can open a PR including the steps you executed to run the program. Thank you.

**Comment by anddann** [comment]
@zhaolida98 I've fixed the bug in the branch. If it works for you please let me know. Then I'll merge the PR.

**Comment by zhaolida98** [comment]
I have tested. Everything works perfectly! I think you can merge the PR. Thanks again.

**Comment by anddann** [comment]
You are welcome! I'm happy to hear when you are done and to read your paper. Please keep me updated!
